esp/logging/logginglib/logthread.cpp

```diff
@@ -422,7 +423,7 @@ void CLogThread::checkRollOver()
             StringBuffer reqBuf;
             const char* GUID = pEspRequest->getGUID();
             if(GUID && *GUID && serializeLogRequestContent(pEspRequest, reqBuf))
-                logFailSafe->Add(GUID, reqBuf.str(), nullptr);
+                logFailSafe->Add(GUID, nullptr, reqBuf.str(), nullptr);
```

Comment:
If pEspRequest contains scriptValues, should they be propagated instead of passing nullptr?